scripts/wiki_ship_stat_parser.py

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import math
+import sys
+import copy
+import json
+import numpy as np
+import operator                 # for sorted
+import sys
+from pprint import pprint
+
+shiptype = {
+    "heavy_cargo_shuttle" : "Heavy cargo shuttle [[File:heavy_cargo_shuttle.png]]",
+    "heavy_courier" : "Heavy courier [[File:heavy_courier.png]]",
+    "heavy_fighter" : "Heavy fighter [[File:heavy_fighter.png]]",
+    "heavy_freighter" : "Heavy freighter [[File:heavy_freighter.png]]",
+    "heavy_passenger_shuttle" : "Heavy passenger shuttle [[File:heavy_passenger_shuttle.png]]",
+    "heavy_passenger_transport" : "Heavy passenger transport [[File:heavy_passenger_transport.png]]",
+    "light_cargo_shuttle" : "Light cargo shuttle [[File:light_cargo_shuttle.png]]",
+    "light_courier" : "Light courier [[File:light_courier.png]]",
+    "light_fighter" : "Light fighter [[File:light_fighter.png]]",
+    "light_freighter" : "Light freighter [[File:light_freighter.png]]",
+    "light_passenger_shuttle" : "Light passenger shuttle [[File:light_passenger_shuttle.png]]",
+    "light_passenger_transport" : "Light passenger transport [[File:light_passenger_transport.png]]",
+    "medium_cargo_shuttle" : "Medium cargo shuttle [[File:medium_cargo_shuttle.png]]",
+    "medium_courier" : "Medium courier [[File:medium_courier.png]]",
+    "medium_fighter" : "Medium fighter [[File:medium_fighter.png]]",
+    "medium_freighter" : "Medium freighter [[File:medium_freighter.png]]",
+    "medium_passenger_shuttle" : "Medium passenger shuttle [[File:medium_passenger_shuttle.png]]",
+    "medium_passenger_transport" : "Medium passenger transport [[File:medium_passenger_transport.png]]",
+}
+
+manufacturers = {
+    "albr" : "Albr Corp.",
+    "auronox" : "Auronox Corporation",
+    "haber" : "Haber Corporation",
+    "kaluri" : "OKB Kaluri",
+    "mandarava" : "Mandarava-Csepel",
+    "opli" : "OPLI-Barnard Inc."
```

Comment:
I suggest adding a trailing comma into that line. That way if it's formatted using yapf, it will format it vertically (like it is now) rather than in one line.

Reply:
done